server/channels/jobs/jobs.go

```diff
@@ -21,6 +21,19 @@ const (
 	CancelWatcherPollingInterval = 5000
 )
 
+// JobLoggerFields returns the logger annotations reflecting the given job metadata.
+func JobLoggerFields(job *model.Job) []mlog.Field {
```

Comment:
As it turns out, there was already a method on `model.Job` to do exactly this -- `LogClone` adds a few more fields than this, and needed a `! = nil` check for ease-of-use, but otherwise simplifies this considerably. Thank you for pushing back on this!!

Reply:
Can you please paste an example output of such a log message into the PR so I can review it? Thanks!

Reply:
Gah, upon reviewing that I realized it's a point in time snapshot of the `Job`, reflecting a status when the logger was annotated and so leads to confusing events like "job completed" while it's in a `pending` state. I've reverted these changes to keep the logger fields within the `jobs` package. I think it makes sense there because a) there's already one in `model` for more "general" logging, and b) the needs of what to log are specific to the jobs infrastructure (e.g. just a handful of key fields) vs. the entire object.

For reference, an example log now just looks like the following -- basically the same as before this change:
```json
{
  "timestamp": "2023-10-06 18:21:44.385 -03:00",
  "level": "info",
  "msg": "SimpleWorker: Job is complete",
  "caller": "jobs/base_workers.go:101",
  "worker_name": "PostPersistentNotifications",
  "job_id": "gzek6dc4d7fgmk494rfroyegxc",
  "job_type": "post_persistent_notifications",
  "job_create_at": "Oct  6 21:21:41.291"
}
```

Reply:
> I've reverted these changes to keep the logger fields within the `jobs` package. I think it makes sense there because a) there's already one in `model` for more "general" logging, and b) the needs of what to log are specific to the jobs infrastructure (e.g. just a handful of key fields) vs. the entire object.

Isn't a) an argument to move the helper method into `model`? Related code should be close to each other. Given that logging code already exists in `model/job.go`, we should move this helper methods code to it and do the declaration of the logged fields.

Reply:
This particular logging strategy is very much an "implementation detail" and not something I mean for us to maintain as part of our `public` interface. In the future, I'm hoping we can refactor the jobs infrastructure in a way that makes even the need to expose this from `jobs` redundant with workers being handed a logger already ready to go.